src/starlingbuilder/engine/tween/DefaultTweenBuilder.as

```diff
@@ -60,11 +60,21 @@ package starlingbuilder.engine.tween
                     if (tweenData is Array)
                     {
                         for each (var item:Object in tweenData)
-                            createTweenFrom(obj, item);
+                        {
+                            if (!trigger
```

Comment:
Lets write this in one line.
Also, the second condition can be replaced with item.triggerName == trigger.